jenkins/bootstrap_test.py

```diff
@@ -732,6 +732,26 @@ def Check(job, name):
 
         self.CheckBootstrapYaml('job-configs/bootstrap-maintenance.yaml', Check)
 
+    def testBootstrapMaintenanceCIYaml(self):
+        def Check(job, name):
+            job_name = 'maintenance-ci-%s' % name
+            self.assertEquals(job_name, job.get('job-name'))
+            self.assertIn('frequency', job)
+            self.assertIn('repo-name', job)
+            self.assertIn('.', job['repo-name'])  # Has domain
```

Comment:
Factored some of this out (although I generally try to minimize abstractions in tests. I generally try to optimize the ease of understanding over the number of lines.